Backpack/TappableLinkLabel/Classes/BPKTappableLinkLabel.m

```diff
@@ -73,6 +73,7 @@ - (void)setupWithStyle:(BPKFontStyle)style {
     _persistedLinks = [[NSMutableArray alloc] init];
     _fontStyle = style;
     _contentView = [[TTTAttributedLabel alloc] initWithFrame:CGRectZero];
+    _contentView.extendsLinkTouchArea = YES;
```

Comment:
This used to be YES by default. Keeping it the same as before to ensure accessibility.